src/DittoMachine.sol

```diff
@@ -316,12 +318,23 @@ contract DittoMachine is ERC721, ERC721TokenReceiver, ERC1155TokenReceiver, Clon
             // buying out the previous clone owner
             address curOwner = ownerOf[cloneId];
             uint256 subsidyDiv2 = subsidy >> 1;
+
+            {
+                address token = _ERC20Contract;
```

Comment:
we can use `_ERC20Contract` directly.

Reply:
did this because i get a stack too deep when I tried that